README.md

```diff
@@ -10,28 +10,42 @@ requirements. This repository contains:
 * Usage instructions
 
 ## Prerequisites
+
 * An account in an OpenShift 3.2+ environment and a project
 
-* Gogs requires a database to store its information. Provisioning a database is
-  out-of-scope for this repository. If you wish to run the database on
-  OpenShift, it is suggested that you deploy PostgreSQL using persistent
-  storage. More information on the OpenShift PostgreSQL deployment is here:
+## Deployment
+Gogs can be easily deployed using the included templates in `openshift` namespace.
```

Comment:
This would require putting the template into the `openshift` namespace, which isn't a default for either Origin or OCP AFAIK.